guides/common/modules/proc_creating-an-activation-key.adoc

```diff
@@ -4,10 +4,6 @@
 You can use activation keys to define a specific set of subscriptions to attach to hosts during registration.
 The subscriptions that you add to an activation key must be available within the associated Content View.
```

Comment:
I see the previous comment was marked resolved, but this is still not deleted. Need to remove these sentences:

```
You can use activation keys to define a specific set of subscriptions to attach to hosts during registration.
The subscriptions that you add to an activation key must be available within the associated Content View.
```